tests/smoke/smoke_test.go

```diff
@@ -0,0 +1,70 @@
+package smoke_test
+
+import (
+	"encoding/json"
+	"fmt"
+	"os/exec"
+	"time"
+
+	"github.com/accuknox/auto-policy-discovery/src/types"
+	"github.com/kubearmor/KubeArmor/tests/util"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/rs/zerolog/log"
+)
+
+var _ = BeforeSuite(func() {
+	//install wordpress-mysql app
+	err := util.K8sApply([]string{"res/wordpress-mysql-deployment.yaml"})
+	Expect(err).To(BeNil())
+	time.Sleep(5 * time.Second)
+
+	// install wordpress-mysql app
+	_, err = util.Kubectl(fmt.Sprintf("apply -f https://raw.githubusercontent.com/kubearmor/discovery-engine/dev/deployments/k8s/deployment.yaml"))
+	Expect(err).To(BeNil())
+	time.Sleep(5 * time.Second)
+
+	// delete all KSPs
+	err = util.DeleteAllKsp()
+	Expect(err).To(BeNil())
+
+	// enable kubearmor port forwarding
+	err = util.KubearmorPortForward()
+	Expect(err).To(BeNil())
+})
+
+var _ = AfterSuite(func() {
+	util.KubearmorPortForwardStop()
+})
+
+func discover(ns string, l string) (types.KubeArmorPolicy, error) {
+	var policy types.KubeArmorPolicy
+	cmd, err := exec.Command("karmor", "discover", "-n", ns, "-l", l, "-f", "json").Output()
+	if err != nil {
+		log.Error().Msgf("Failed to apply the `karmor discover` command : %v", err)
+	}
+	err = json.Unmarshal(cmd, &policy)
+	if err != nil {
+		log.Error().Msgf("Failed to unmarshal the policy : %v", err)
+	}
+	return policy, err
+}
+
+var _ = Describe("Smoke", func() {
```

Comment:
before running a test, make sure that `discovery-engine` and `wordpress-app` is up and running using `BeforeEach()`